lib/get-format.js

```diff
@@ -18,7 +18,9 @@ const extensionFormatMap = {
   '.cjs': 'commonjs',
   '.js': 'module',
   '.json': 'json',
-  '.mjs': 'module'
+  '.mjs': 'module',
+  // Taken from https://github.com/nodejs/node/blob/595ce9dac63cdd32d893c3319328a752719644d2/lib/internal/modules/esm/formats.js#L28-L30
+  '.wasm': 'wasm'
```

Comment:
```suggestion
```

Please drop it for now. The `experimentalWasmModules` flag is not supported (see readme). If needed, we can add it later.

Reply:
Given that `.ts` etc can now also be loaded, without it being in the map, my guess is also it that it doesn’t have to be added and the test you added will work fine even without it?